src/components/surveys/survey-design/SurveyDesign.tsx

```diff
@@ -23,11 +23,12 @@ import {SURVEY_ICONS} from '../widgets/SurveyIcon'
 import IntroInfo from './IntroInfo'
 import AddQuestionMenu from './left-panel/AddQuestionMenu'
 import LeftPanel from './left-panel/LeftPanel'
-import QUESTIONS, {QuestionTypeKey} from './left-panel/QuestionConfigs'
+import QUESTIONS, {QuestionTypeKey, getQuestionId} from './left-panel/QuestionConfigs'
 import QuestionEditPhone from './question-edit/QuestionEditPhone'
 import QuestionEditRhs from './question-edit/QuestionEditRhs'
 import QuestionEditToolbar from './question-edit/QuestionEditToolbar'
 import ReadOnlyBanner from '@components/widgets/ReadOnlyBanner'
+import { ConnectWithoutContactOutlined } from '@mui/icons-material'
```

Comment:
Just curious, are the two new imports (`getQuestionId`, `ConnectWithoutContactOutlined`) needed? I didn't seem them used in the changes, but I might've missed them!